outline.md

```diff
@@ -0,0 +1,59 @@
+## Growing an Inner Source Program
+
+45 min
+
+## Short Description
+
+"Inner source is a great idea! Go do inner source!"
+You're the dedicated champion for inner source in your company.
+What do you do now?
+How do you realistically affect the behavior of dozens or hundreds of teams to the point where robust and pervasive inner sourcing is a normal part of the way that engineering is done?
+
+This presentation shares principles, practical anecdotes, and relatable examples on this situation gleaned from experience at Nike.
+While not claiming all of the answers, it is insightful summary of over a year of work in the area.
+
+## Audience
+This presentation will be given at [the ISC Spring Summit](http://paypal.github.io/InnerSourceCommons/events/isc-spring-2018-agenda/).
+It is intended for those that are evangelizing inner source in a specific company.
+
+## Detailed description
+
+#### Outline
+
+An inner source program is the set of tools, documentation, training, and so forth provided so that any team can participate in inner source effectively as either a maintainer or contributor.
+To frame the talk, we'll explain the goals of an inner source program as well as the context within-which we've begun to pursue those goals.
+Even though the learning in this presentation is occurring in that context, we suspect that much of what's here applies generally.
+
+When building an inner source program you should have a vision of what services you will provide to facilitate inner sourcing _as well as_ a strategy you will employ to roll out and make effective these services in your organization.
+We'll explain principles, resources, and specific examples that can aid and guide in doing so.
+
+The services of the inner source program should provide two things.
+First, they must teach, define, and document the type of culture that is conducive to inner source.
+Second, there must be software that reinforces that culture while simultaneously lowering the barrier to entry to participating in that culture.
+Examples of these services include:
```

Comment:
Not sure that you should provide this many examples.  I think they dilute the first two points a bit.  I think the only one we're super confident in is the playbook model.  And this facilitates this point "Software/convention to facilitate individuals communicating with one another"

I'd say that Curated training on the above software and documentation is what takes things to the next level.  Not just having a program but ensuring that it thrives.

Reply:
I think the thing missing here is the regular customer outreach and marketing that goes along with it.   I think without a regular pulse you'll end up with the flash mob problem, where the point to point connections are only as good as the person network who is facilitating the communities.   I know that we haven't defined it but it is critical to the success we've had so far.  If you stopped doing we wouldn't have our communities.  So I think you need to be able to articulate this point in your talk.  Without it you'd have people mandated to participate, which would result in a disastrous innersource program.

Reply:
> Not sure that you should provide this many examples.  I think they dilute the first two points a bit. I think the only one we're super confident

Is the recommendation to leave them out due to the dilution angle or lack of confidence angle?  If dilution, these points are meant to be specific examples of the two points (expand on them).  If lack of confidence, I think it's OK as long as it's the kind of stuff we think we want to do.  The point (in my mind) of the presentation is to present a case for working this way so that as we do it in the upcoming months others may be interested in working with us as we go about implementing these ideas.

Reply:
> missing here is the regular customer outreach

I think that would go further below in the section of how to roll out the program?

Reply:
My main concern is that by giving such concrete examples folks will think that they should implement the same things that we have to get an innersource program up and running.  I don't think we're ready to give that level of recommendation. 

Instead I'd propose that you focus on describing how to create the culture that will cut through [strategy](https://nikedigital.slack.com/archives/C6TC8J68L/p1525311502000026).

Reply:
>  The point (in my mind) of the presentation is to present a case for working this way so that as we do it in the upcoming months others may be interested in working with us as we go about implementing these ideas.

it seems like that should be at the end then as your call to action.  Here are some of the solutions we've started pursuing....

Reply:
Sounds good.  There is a sentence at the end about hoping to work together.